pylint/checkers/base/docstring_checker.py

```diff
@@ -73,6 +73,11 @@ class DocStringChecker(_BasicChecker):
             "docstring.",
             {"old_names": [("C0111", "missing-docstring")]},
         ),
+        "C0150": (
+            "%s",
+            "fixme-in-docstring",
+            "Used when a warning note as FIXME or XXX is detected in docstring.",
```

Comment:
Imo should be a rational (why it's bad) and not when it's raised. (I know the current descriptions are a bad example of that.)